.github/workflows/on_pr.yml

```
@@ -56,6 +56,70 @@ jobs:
           name: report_light_ci
           path: ./report_______light_ci.md
 
+  # Hang while any push/schedule jobs in the repo are queued or in_progress.
+  # Runs after CI/metrics, then "all" depends on it. Gives up with success
+  # after 1 hour.
+  wait_for_push_schedule:
+    name: Wait for push/schedule jobs
+    needs: [ main_workflow, upload_metrics_history ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)